### PR TITLE
feat: space modals in mantine

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: 

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/ReactHookForm/RadioGroup.tsx
Comment (magnew) [inline packages/frontend/src/components/ReactHookForm/RadioGroup.tsx:1]: 🎉

File: packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx [hunk 1]
```diff
@@ -38,26 +28,35 @@ const renderUser = (user: {
...
-}) => {
-    return (
-        <FlexWrapper key={user.userUuid}>
-            <UserCircle>
-                {user.firstName.substr(0, 1) + user.lastName.substr(0, 1)}
-            </UserCircle>
+}) => (
+    <Group spacing="sm" position="apart" noWrap>
+        <Group>
+            <Avatar radius="xl" tt="uppercase" color="blue">
```
Comment (magnew) [inline packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx:34]: Are these looking blue at all to you? I see them as grey even though they are set to blue here.

**Review by magnew — APPROVED** [review]
Looks great! Tested and everything seems to work. 

One minor issue I found in test, not really a bug but a little confusing: when are creating a space and don't set a name, the create button on the permissions section is active, but does nothing. We could either disable 'Continue' if there is no name, or disable create.

I'm fine with doing this in a follow up if you want. 
 
Here's what I mean

Kapture 2023-12-19 at 15 16 02

File: packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx:34]: can you take a screenshot? there are few places where I had to update colors

Comment (magnew) [reply packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx:34]: It seems like everywhere I see these avatars they are gray. Whatever their access.

Screenshot 2023-12-19 at 16 02 02

Comment (IrakliJani) [reply packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx:34]: must be render not picking up the changes. triggering manual sync!

**Pull request merged**